docker/entrypoint.sh

```diff
@@ -1,11 +1,5 @@
 #!/bin/bash
 
 set -e
-pip install -r requirements.txt -c constraints.txt
-pip install -e /pace-util -c constraints.txt
-pip install -e /fv3core -c constraints.txt
-pip install -e /fv3gfs-physics -c constraints.txt
-pip install -e /stencils -c constraints.txt
-pip install -e /dsl -c constraints.txt
-pip install -e /driver -c constraints.txt
+./install.sh
```

Comment:
💯

Reply:
oh wait, I think we cannot change this because those folders are mounted inside docker for when we run CI on gce.